### ISSUE TITLE
Date Modified"

### ISSUE THREAD

**Issue by rieje** [cboxdoerfer/fsearch#75]
Column-resizing feels clunky and limited. I

Also, ideally, the columns can be moved to different positions.

**Comment by cboxdoerfer** [comment]
I agree and I too don't like the current column behavior at all. Unfortunately some of the issues are caused by the GtkTreeView widget and GtkTreeViewColumn and I'm not sure if I can fix those. But I'll give it a try.

> Lastly, ideally, the columns can be moved re-organized to different positions.

That's already on my very long todo list, but it should be easy to implement and will happen at some point.

**Comment by smjure** [comment]
This problem seems to be still present. Here is a pic from the latest release, where the denoted area with red is "Date Modified".  

![image](https://user-images.githubusercontent.com/22912191/100970008-5ecd0c80-3534-11eb-8500-a33c79c651bc.png)

Would be nice if this is possible to fix.

**Comment by smjure** [comment]
Perhaps temporary solution would be to put the "Type" column as the last one as it's much less important (I believe to most of users) than the "Date Modified". Thanks :)
